osf/migrations/0135_add_ember_auth_register_waffle_flag.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-09-14 15:12
+from __future__ import unicode_literals
+from waffle.models import Flag
+from django.db import migrations
+
+def reverse_func(state, schema):
+    Flag.objects.get(name='ember_auth_register').delete()
+    return
+
+def add_ember_auth_register_waffle_flag(state, schema):
```

Comment:
We've stopped adding flags in migrations, since it's unnecessary. Can you remove this migration?

Reply:
Mm..but we might change our policy on this.

Discussing with @aaxelb in regards to #8725 . Hold tight.

Reply:
OK, so we decided to use migrations for adding new flags.

Abram added a custom migration command, `AddWaffleFlags` for doing this. Can you rebase this on top of develop and remake this migration using that command?